File: python/5086.py
def main(a, b) :
    if (b%a==0):
        return "factor"
    elif (a%b==0) :
        return "multiple"
    else :
        return "neither"

while True :
    a, b = list(map(int, input().split()))
    if (a == 0 & b == 0) : break
    print(main(a, b))

File: algorithm/src/5000~10000/5086.kt
package `5000~10000`

fun main() {
    while (true) {
        val (a, b) = readLine()!!.split(" ").map { it.toInt() }
        if (a == 0 && b == 0) break
        println(when {
            b % a == 0 -> "factor"
            a % b == 0 -> "multiple"
            else -> "neither"
        })
    }
}
